__test__/App.spec.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { BrowserRouter, Switch } from 'react-router-dom';
 import thunk from 'redux-thunk';
+import { mount } from 'enzyme';
```

Comment:
Hi @tolumide-ng and @olorunwalawrence,

Please remove the `mount` function since it has been configured globally on the project.

Reply:
Okay